src/Command/DrupalUpdate/DrupalPackageManager.php

```diff
@@ -145,7 +145,7 @@ public function getFileInfo(string $filepath, string $drupal_project_cwd_path):
       'project',
     ];
     $info_extension_file = @parse_ini_file($filepath, FALSE, INI_SCANNER_RAW);
-    if (is_bool($info_extension_file) && $info_extension_file == FALSE) {
+    if (!$info_extension_file) {
```

Comment:
I meant `is_bool($info_extension_file) && !$info_extension_file` only, not to remove the boolean check.